SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -643,10 +643,7 @@ void USpatialSender::SendAuthorityIntentUpdate(const AActor& Actor, VirtualWorke
 	// This should always happen with USLB.
 	NetDriver->LoadBalanceEnforcer->ShortCircuitMaybeRefreshAuthorityDelegation(EntityId);
 
-	if (NetDriver->SpatialDebugger != nullptr)
-	{
-		NetDriver->SpatialDebugger->ActorAuthorityIntentChanged(EntityId, NewAuthoritativeVirtualWorkerId);
-	}
+	NetDriver->SpatialDebuggerSystem->ActorAuthorityIntentChanged(EntityId, NewAuthoritativeVirtualWorkerId);
```

Comment:
check against null here

Reply:
Doesn't seem to be possible as it's a `TUniquePtr` initialized in `CreateAndInitializeCoreClasses`; do you think a `check(NetDriver->SpatialDebuggerSystem.IsValid())` would work?